app/views/devise/sessions/_mfa_enrolment.erb

```diff
@@ -2,28 +2,32 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-one-half">
-
-      <%= f.label :totp_code, class: "govuk-label" %>
-      <%= f.text_field :totp_code, autofocus: true, autocomplete: 'off', class: "govuk-input govuk-input--width-10" %>
-      <%= f.hidden_field :email, value: { email: false } %>
-
+      <div class="govuk-!-margin-bottom-7">
```

Comment:
the `govuk-!-margin-bottom-7` modifier shouldn't be needed

Reply:
<img width="357" alt="Screen Shot 2019-10-17 at 15 13 14" src="https://user-images.githubusercontent.com/41922771/67016887-d3ef3f80-f0f0-11e9-9e2d-d4571dad1182.png">

**WITHOUT MARGIN**

I just added it because I considered that the two elements were too close together without a margin. Happy to remove though

<img width="326" alt="Screen Shot 2019-10-17 at 15 13 08" src="https://user-images.githubusercontent.com/41922771/67016886-d3ef3f80-f0f0-11e9-8be7-d00deb51090e.png">

**WITH MARGIN**

Reply:
if you take out the QR code of the that new div, and move the text/details part (Unable to scan) into `<p class="govuk-body">` element it should make it nicer...